paperscraper/tests/test_pdf.py

```diff
@@ -166,13 +166,30 @@ def test_save_pdf_from_dump(self):
         save_pdf_from_dump(TEST_FILE_PATH, pdf_path=SAVE_PATH, key_to_save="doi")
         shutil.rmtree(SAVE_PATH)
 
+    def test_api_keys_none(self):
+        test_doi = {"doi": "10.1038/s41587-022-01613-7"}  # Use a DOI known to be in PMC
+        save_pdf(test_doi, filepath=SAVE_PATH, api_keys=None)
+
+    def test_api_key_file(self):
+        test_doi = {"doi": "10.1002/smll.202309431"}  # Use a DOI known to be in PMC
```

Comment:
The smll DOI is not in PMC but from Wiley.
```suggestion
        test_doi = {"doi": "10.1002/smll.202309431"}  # Use a DOI from Wiley
```